src/components/Staking/StakingStatsBox.tsx

```diff
@@ -177,7 +170,16 @@ const StakingStatsBox: React.FC<IProps> = () => {
         )}
         <Label>
           <Translation id="page-staking-stats-box-metric-1" />
-          <BeaconchainTooltip />
+          <BeaconchainTooltip>
+            <div>
+              <Text>
+                Total cumulative ETH at stake on the Beacon Chain (does not
+                include accumulated rewards above 32 ETH)
```

Comment:
Not blocking but this box is a bit clunky when it has a large amount of text.

![Screenshot 2023-03-22 at 16 27 55](https://user-images.githubusercontent.com/62268199/226972614-e9c552c3-7336-4329-b754-e457a46ea4be.png)

Reply:
I honestly don't think it's well understood that rewards above 32 don't contribute to this metric, especially since in the past we've used a different endpoint/calculation for this which _did_ include the total amount of ETH on the Beacon Chain (including rewards).

Perhaps a similarly concise message, but noting 32:

```suggestion
                Total cumulative ETH at stake on the Beacon Chain, not including rewards over 32
```

Reply:
or "balances" instead of "rewards"... since "rewards" can be a bit ambiguous... (ie. I was offline a while, dipped to 31 ETH... then went back online and earned "rewards"... but those would still be active stake until going above 32)

```suggestion
                Total cumulative ETH at stake on the Beacon Chain, not
                including balances over 32
```